### ISSUE TITLE
HALT PowerLVS Validation Errors

### ISSUE THREAD

**Issue by jweber-ice** [WIPACrepo/stf#61]

**Comment by jweber-ice** [comment]
Data saved to cobalt.icecube.wisc.edu:/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14

```
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183830/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    "name": "chan_00",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183830/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183830/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183830/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                        "10 <= x <= 1000"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183830/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183830/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 1023.0

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180832/degg-0c52f6f87fbd/PowerLVS-base-v1.0_degg-0c52f6f87fbd.json-                    "name": "chan_04",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180832/degg-0c52f6f87fbd/PowerLVS-base-v1.0_degg-0c52f6f87fbd.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180832/degg-0c52f6f87fbd/PowerLVS-base-v1.0_degg-0c52f6f87fbd.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180832/degg-0c52f6f87fbd/PowerLVS-base-v1.0_degg-0c52f6f87fbd.json-                        "10 <= x <= 1000"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180832/degg-0c52f6f87fbd/PowerLVS-base-v1.0_degg-0c52f6f87fbd.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180832/degg-0c52f6f87fbd/PowerLVS-base-v1.0_degg-0c52f6f87fbd.json-                    "measured_value": 9.8438

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180835/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    "name": "chan_04",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180835/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180835/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180835/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                        "10 <= x <= 1000"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180835/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180835/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 8.0781

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183830/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    "name": "chan_00",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183830/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183830/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183830/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                        "10 <= x <= 1000"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183830/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183830/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 1023.0

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191806/degg-0c52f6f87fbd/PowerLVS-base-v1.0_degg-0c52f6f87fbd.json-                    "name": "chan_04",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191806/degg-0c52f6f87fbd/PowerLVS-base-v1.0_degg-0c52f6f87fbd.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191806/degg-0c52f6f87fbd/PowerLVS-base-v1.0_degg-0c52f6f87fbd.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191806/degg-0c52f6f87fbd/PowerLVS-base-v1.0_degg-0c52f6f87fbd.json-                        "10 <= x <= 1000"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191806/degg-0c52f6f87fbd/PowerLVS-base-v1.0_degg-0c52f6f87fbd.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191806/degg-0c52f6f87fbd/PowerLVS-base-v1.0_degg-0c52f6f87fbd.json-                    "measured_value": 8.0312

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_195713/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    "name": "chan_04",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_195713/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_195713/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_195713/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                        "10 <= x <= 1000"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_195713/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_195713/degg-2f8c8eef6792/PowerLVS-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 8.1719
```

**Comment by jweber-ice** [comment]
Updated pass/fail per HALT results. Closed by commit d71c1214f016f45a8818c146206e89746ef1ba38
